tesseract/tesseract_collision/src/bullet/bullet_cast_bvh_manager.cpp

```diff
@@ -166,7 +166,14 @@ bool BulletCastBVHManager::enableCollisionObject(const std::string& name)
   if (it != link2cow_.end())
   {
     it->second->m_enabled = true;
-    link2castcow_[name]->m_enabled = true;
+    if (it->second->getBroadphaseHandle())
+      broadphase_->getOverlappingPairCache()->cleanProxyFromPairs(it->second->getBroadphaseHandle(), dispatcher_.get());
+
+    auto cast_cow = link2castcow_[name];
+    cast_cow->m_enabled = true;
+    if (cast_cow->getBroadphaseHandle())
+      broadphase_->getOverlappingPairCache()->cleanProxyFromPairs(cast_cow->getBroadphaseHandle(), dispatcher_.get());
+
```

Comment:
Again maybe some comments on what these steps do and why they are necessary

Reply:
Done